src/style.css

```diff
@@ -1 +1,65 @@
-/* Styles go here */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo__img {
+  width: 40px;
+  height: 40px;
+  display: flex;
+}
+
+.nav__list {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  cursor: pointer;
```

Comment:
Why you add cursor pointer for the whole list? Cursor pointer already applied to every .nav__link element because it is a link and it's apply by default